### PR TITLE
fixes #528 SABNZB Path calculates mylar address incorrect

### LINKED ISSUE #528: Sabnzb is no longer able to download the packages
**Describe the bug**
Sabnzb is no longer able to download the packages.
The reason for this is that search.py line 4126, 4139, 4150, 4155 use HTTP_ROOT instead of the Port.

**Additional information**
I will provide a PR shortly to fix the issue

### PR DESCRIPTION
**Pull request by miracle152005**
SABNZB was no longer able to download because the address to get the nzb from was not correct.

### CODE REVIEW

File: mylar/search.py [hunk 1]
```diff
@@ -4147,12 +4147,12 @@ def searcher(
                             mylar_host = (
                                 '%s%s:%s%s'
                                 % (proto, mylar.CONFIG.HTTP_ROOT,
```
Comment (evilhero) [inline mylar/search.py:4149]: This should be ``` % (proto, mylar.CONFIG.HTTP_HOST, mylar.CONFIG.HTTP_PORT, hroot)```

**Review by evilhero — CHANGES_REQUESTED** [review]
Stupid typos during flaking.

Just need that one correction that I mentioned, and I'll push this. 

Thanks for bringing it to our attention.

File: mylar/search.py (hunk 1 above)
Comment (miracle152005) [reply mylar/search.py]: Did not see that one.
Should be fixed now

**Review by evilhero — APPROVED** [review]
good to go!

**Pull request merged**